src/kontrol/prove.py

```diff
@@ -864,14 +872,15 @@ def _init_cterm(
         init_subst.update(init_subst_test)
     else:
         accounts: list[KInner] = []
+        contract_account_id = Foundry.symbolic_contract_prefix() + '_' + contract_name
```

Comment:
why not `Foundry.symbolic_contract_id(contract_name)` instead?
```suggestion
        contract_account_id = Foundry.symbolic_contract_id(contract_name)
```

Reply:
Because that will already have `_ID`, and then the `symbolic_account` would add `_ID`, `_BAL`, etc. to that. That, however, indicates that the `symbolic_contract_id` is not defined well.

Reply:
I've now differentiated between `symbolic_account_name` and `symbolic_account_id`.